### PR TITLE
Fixes #57

### LINKED ISSUE #57: Largest_Sum_Subarray
Aim : Given an array arr of N integers. Find the contiguous sub-array with maximum sum.
Expected Time Complexity: O(n)
Expected Auxiliary Space: O(1)

I want to write this code using Window Sliding Technique in C++ Language, please assign it to me.

### PR DESCRIPTION
**Pull request by neilotpal**

### CODE REVIEW

File: Array/Largest_sum_subarray.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* Given an array of positive and negative integers. We have to find subarray having maximum sum */
+#include<iostream>
+using namespace std;
+
+// Function to find the maximum sum of a contiguous subarray
+int kadaneAlgo(int arr[], int n)
+{
+    // stores the maximum sum subarray found so far
+    int max_so_far = INT_MIN;
```
Comment (sakshigoel19) [inline Array/Largest_sum_subarray.cpp:9]: ‘INT_MIN’ was not declared in this scope. Include limits.h header file to resolve this error.

Comment (neilotpal) [reply Array/Largest_sum_subarray.cpp:10]: Recheck please, updated the code

**Review by sakshigoel19 — APPROVED** [review]

**Review by v-vanisha — CHANGES_REQUESTED** [review]
Hey, according to the guidelines, you need to give a brief description of the algorithm that you are applying to your code. Kindly, do the same!

**Comment by neilotpal** [comment]
Given a brief description, recheck please.

**Review by sakshigoel19 — APPROVED** [review]

**Review by v-vanisha — APPROVED** [review]

**Review by Vanshikagarg17 — APPROVED** [review]

**Pull request merged**